cmd/edenStart.go

```diff
@@ -39,6 +39,12 @@ var startCmd = &cobra.Command{
 			adamPort = viper.GetString("adam.port")
```

Comment:
In general I'd like to discuss to have errors in pkg and don't have them in cmd... Discussable

Reply:
Yes, maybe for some errors this may be reasonable -- let's discuss later.